.github/workflows/test.yml

```diff
@@ -49,13 +49,14 @@ jobs:
     - uses: actions/download-artifact@v3
       with:
         name: mnelab-wheel
-    - name: Install Linux packages
+    - name: Debug
       run: |
-        sudo apt update
-        sudo apt install libxcb-icccm4 libxcb-image0 libxcb-keysyms1 \
-                         libxkbcommon-x11-0 xvfb libxcb-randr0 \
-                         libxcb-render-util0 libxcb-xinerama0 libegl1 \
-                         libxcb-shape0
+        set -e
+        LD_DEBUG=libs python -c "from PySide6.QtWidgets import QApplication, QWidget; app = QApplication([]); import matplotlib; matplotlib.use('QtAgg'); import matplotlib.pyplot as plt; plt.figure()"
```

Comment:
The error log says:
```
ModuleNotFoundError: No module named 'PySide6'
```
This step needs to go after PySide6 has been installed, which would be after the "Install Python Packages" step

Reply:
Got it. New error: https://github.com/cbrnr/mnelab/actions/runs/4768501192/jobs/8477907996?pr=372#step:7:852